tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -423,7 +424,7 @@ def _get_reader_for_run(self, run):
     reader = None
     if config.model_checkpoint_path and _using_tf():
       try:
-        reader = tf.compat.v1.pywrap_tensorflow.NewCheckpointReader(
+        reader = _pywrap_tensorflow.NewCheckpointReader(
```

Comment:
I think we can actually just change this to `tf.compat.v1.train.NewCheckpointReader` if you don't mind.  It looks like `tf.train.NewCheckpointReader` in TF 1.x is the place this symbol was properly exposed:

https://www.tensorflow.org/api_docs/python/tf/train/NewCheckpointReader

Reply:
I changed it to tf.train.load_checkpoint. It seems to be the same thing:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/checkpoint_utils.py#L43

Reply:
Even better, thank you!